include/glow/Quantization/Base/Base.h

```diff
@@ -99,6 +101,10 @@ QuantizationTransform32To8 quantizeScaleOffset32To8(float scale,
 TensorQuantizationParams chooseQuantizationParams(float min, float max,
                                                   Schema schema = Asymmetric);
 
+/// \return a mapping from the \p inTy to the \p outTy given the function \p f.
```

Comment:
based on the return result this seems to imply that it works for quantized int8 only. can you reflect that in the comment?